test/JDBC/expected/temp_table_rollback-vu-prepare.out

```diff
@@ -0,0 +1,55 @@
+CREATE VIEW enr_view AS
```

Comment:
You may also need to add tests for other isolation types such as `xact_abort` and `snapshot`